far/filelist.cpp

```diff
@@ -982,7 +982,7 @@ long long FileList::VMProcess(int OpCode,void *vParam,long long iParam)
 					if (i.empty())
 						continue;
 
-					const auto Pos = FindFile(PointToName(i), true);
+					const auto Pos = FindFile(i, !i.contains(path::separator));
```

Comment:
Sorry, just realized this won't compile too (or at least not everywhere).

`!i.contains(path::separator)` -> `!contains(i, path::separator)`